src/modules/csm/checkpoint.py

```diff
@@ -202,6 +204,9 @@ def _check_duty(
             self.state.add_processed_epoch(duty_epoch)
             self.state.commit()
             self.state.status()
+            unprocessed_epochs = self.state.unprocessed_epochs
```

Comment:
Why so?

Reply:
to do calcs once and be sure that the value will be the same for both of metrics

Reply:
Let's do it outside lock?

Reply:
self.state.status() seems can be done outside